### PR TITLE
Cleanup arbiter.c

### PR DESCRIPTION
**Pull request by movrsi**
- Remove a bunch of newlines that made no sense.
- Slightly optimize branching in ARB_Clock.
- Left a comment so the maintainer can decide on inlining certain routines.

### CODE REVIEW

File: arbiter.c [hunk 1]
```diff
@@ -73,6 +73,11 @@ void ARB_UpdateDelays(arbiter_t *chip, uint64_t cycles)
     chip->input.d8_out = DELAY_Update(&chip->d8, cycles, chip->input.ext_m3);
 }
 
+/*
```
Comment (nukeykt) [inline arbiter.c:76]: comments are not needed

File: arbiter.c [hunk 2]
```diff
@@ -667,33 +535,42 @@ void ARB_Clock(arbiter_t *chip)
...
+        chip->w128 = (chip->input.ext_vaddress_in & 0x7fff80) != 0x508000; // IO chip
+        chip->w122 = (chip->input.ext_vaddress_in & 0x7ffe00) != 0x508800; // Z80 ctrl
+        chip->w129 = (chip->input.ext_vaddress_in & 0x7fff80) != 0x509000; // FDC
+        chip->w124 = (chip->input.ext_vaddress_in & 0x7fff80) != 0x509800; // TIME
+        chip->ext_ia14 = !chip->va14_in; // IA14
+        chip->w86 = (chip->input.ext_zaddress_in & 0xff00) != 0x7f00; // VDP
+        chip->w136 = !((chip->input.ext_zaddress_in & 0xc000) == 0 && !chip->w220); // Z80 RAM
+        chip->w140 = !((chip->input.ext_zaddress_in & 0xe000) == 0x4000 && !chip->w220); // YM3438
+        chip->w150 = !((chip->input.ext_zaddress_in & 0xff00) == 0x6000 && !chip->input.ext_zwr_in && !chip->w220); // z80 bank
+    }
```
Comment (nukeykt) [inline arbiter.c:573]: variables are not set when `ext_m3` is true

**Review by nukeykt — CHANGES_REQUESTED** [review]

File: arbiter.c (hunk 2 above)
Comment (movrsi) [reply arbiter.c:573]: Hm, i'm checking if its false, did i reverse the logic by accident?

Comment (nukeykt) [reply arbiter.c:573]: something like this I mean
```
if (ext_m3)
{
var = blah, blah;
}
else
{
var = 1;
}
```

Comment (movrsi) [reply arbiter.c:573]: Ah i see, ill resolve the following issues.

**Comment by movrsi** [comment]
Resolved; while i am at it, would you mind if certain routines were converted into 

```C
static inline void```

as the `ARB_Clock` routine calls them quite a bit, it would keep code readable but also provide some optimization benefits for the arbiter.

**Comment by nukeykt** [comment]
yea, good point

**Comment by movrsi** [comment]
Alright, ill resolve that quickly as well.

**Comment by nukeykt** [comment]
thx

**Pull request merged**